### ISSUE TITLE
Version 2.0.1 hasn't be built before being published to npm

### ISSUE THREAD

**Issue by jue89** [alexeden/spi-node#1]
Hey @alexeden,

Thank you for your library! I like the interface and the usage of N-API :)

I think you forgot to execute `npm run build` before you published this lib to npm. The transpiled JS version of your library stuck in major version 1. In contrast, the TS version is up-to-date.

You may extend you package.json:
```js
{
  ...
  "scripts": {
    "prepublishOnly": "rimraf dist && tsc -p src",
    ...
  },
  ...
}
```

**Comment by alexeden** [comment]
Doh! Published a fix as 2.0.3. Let me know if that fixes it for you. Thanks for pointing it out!

**Comment by jue89** [comment]
Looks great! :)
Thank you!
